packages/react-components/src/Concepts/Accessibility/AccessibleComponents.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/addon-docs';
+import LinkTo from '@storybook/addon-links/react';
+import { ScenariosListLink } from '../../AccessibilityScenarios/utils';
+
+<Meta title="Concepts/Developer/Accessibility/Components" />
+
+## Components accessibility
+
+Fluent UI components are designed to support various aspects of accessibility by default, so that they can be used with different input methods (mouse, touch, keyboard, screen readers) as well as fulfill different rendering and layout requirements (theming, zoom, contrast).
+
+The [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) and [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) are community-driven standards for accessible web pages and applications. They provide a basic framework allowing consistent usage of web pages and applications with or without assistive technologies.
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them.
+
+Using components themselves does not guarantee the application or a page to be accessible. Please see [Accessible Experiences](?path=/docs/concepts-developer-accessibility-experiences--page) page for further points on achieving good level of usability.
+
+## Scope
+
+Fluent UI components fulfill following requirements:
+
+- **DOM structure** that provides **semantical value** either by using correct element types or roles
```

Comment:
`semantical` is what I would have grown up speaking, I don't think our american colleauges know about this one though 😉
```suggestion
- **DOM structure** that provides **semantic value** either by using correct element types or roles
```